docs/pages/management/register-services/register-services-join-token.mdx

```diff
@@ -0,0 +1,253 @@
+---
+title: Register Services via Join Token
+description: How to register services with your Teleport cluster via a join token
+---
+
+In this guide, we will show you how to register a Teleport service with your
+cluster by presenting a **join token**. 
+
+In this approach, you declare your intention to register a new service, and
+Teleport generates a secure token for the service. When the service starts, it
+presents the token, indicating to your Teleport cluster that it is trustworthy.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- A Linux server that you will use to host your Teleport service. 
+
+  In this guide, we will show you how to register a Teleport SSH Service
+  instance. This approach applies to new Proxy Service instances as well as the
+  Kubernetes Service, Database Service, and other Teleport services for
+  accessing resources in your infrastructure.
+
+(!docs/pages/includes/tctl.mdx!)
+
+## Step 1/3. Install Teleport
+
+Install Teleport on the host where you will run your Teleport service.
+
+(!docs/pages/includes/install-linux.mdx!)
+
+## Step 2/3. Join your service to the cluster
+
+In this section, we will join your Teleport service to the Teleport cluster by:
+
+- Obtaining a join token
+- Running your Teleport service with the join token
+
+### Generate a token
+
+Teleport only allows access to services that have joined the cluster.
+
+Once a service joins, it receives a host certificate signed by the cluster's
+Auth Service. To receive a host certificate upon joining a cluster, a new
+Teleport host must present an **invite token**.
+
+An invite token also defines which role a new host can assume within a cluster:
+
+- `proxy`
+- `node`
+- `auth`
+- `app`
+- `db`
+- `kube`
+- `windowsdesktop`
```

Comment:
This list is incomplete. You're missing `bot` and `discovery` (and arguably `instance` too, but I'm not sure it's worth opening that can of worms here)